Firestore/core/src/remote/grpc_connection.cc

```diff
@@ -193,6 +209,12 @@ std::unique_ptr<grpc::ClientContext> GrpcConnection::CreateContext(
   return context;
 }
 
+void GrpcConnection::AddCloudApiHeaders(grpc::ClientContext& context) {
```

Comment:
Note: there are many tests that create a `RemoteStore` but not `FIRFirestore`; they would have the language header blank. I presume this is okay but pointing it out just in case.

Reply:
Why not default to C++? This would make it similar to the way the throw handler defaults to C++.

Reply:
My concern is that if we ever get a bug where the language isn't set for some reason, we would get incorrect overrepresentation of C++ in the data (whereas the numbers should actually add to Objective-C or Unity). What do you think? Also, do you think there should be an assertion to verify that a language was set?

Reply:
While I definitely respect where this concern is coming from, I don't see this as a practical concern. By hooking into class loading in Objective-C there's essentially no way you can use `FIRFirestore` without setting this value to something reasonable. I can it see it being straightforwardly the case that we can make similar guarantees in the other bindings.

Given that we have a number of these kinds of language preferences in play (exception handling, logging, version reporting, and probably others), I'd prefer that they all worked the same way just to make things simpler.

Reply:
The consistency argument SGTM. I added simple logic for reporting the C++ version, PTAL. We might want to change it in the future, but hopefully it's good enough for now.